test/integration/test_messages_api.py

```diff
@@ -0,0 +1,184 @@
+        self.api_instance = messages_api.MessagesApi(api_client)
+        self.account_id = os.environ.get('BW_ACCOUNT_ID')
+
+        # Unauthorized API Client
+        self.unauthorized_api_client = bandwidth.ApiClient()
+        self.unauthorized_api_instance = messages_api.MessagesApi(self.unauthorized_api_client)
+
+        # Message Properties
+        self.application_id = os.environ.get('BW_MESSAGING_APPLICATION_ID')
+        self.to_number = [os.environ.get('USER_NUMBER')]
+        self.from_number = os.environ.get('BW_NUMBER')
+        self.text = 'python integration'
+        self.media = ['https://cdn2.thecatapi.com/images/MTY3ODIyMQ.jpg']
+        self.tag = 'python integration tag'
+        self.priority = PriorityEnum("default")
+
+        # Message Request
+        self.message_request = MessageRequest(
+            application_id=self.application_id,
+            to=self.to_number,
+            _from=self.from_number,
+            text=self.text,
+            media=self.media,
+            tag=self.tag,
+            priority=self.priority,
+        )
+        
+        # Invalid Message Request
+        self.invalid_message_request = MessageRequest(
+            application_id=self.application_id,
+            to=['+invalid'],
+            _from='+invalid',
+            text='',
+        )
+
+
+    def test_create_message(self):
+        response: Message = self.api_instance.create_message(self.account_id, self.message_request, _return_http_data_only=False)
+
+        self.assertEqual(response[1], 202)
+
+        api_response = response[0]
+        self.assertIsInstance(api_response, Message)
+        self.assertEqual(api_response.application_id,self.application_id)
+        self.assertEqual(api_response.to, self.to_number)
+        self.assertEqual(api_response._from, self.from_number)
+        self.assertEqual(api_response.owner, self.from_number)
+        self.assertEqual(api_response.text, self.text)
+        self.assertEqual(api_response.media, self.media)
+        self.assertEqual(api_response.tag, self.tag)
+        self.assertIsInstance(api_response.priority, PriorityEnum)
+        self.assertEqual(api_response.priority, self.priority)
+        self.assertEqual(api_response.segment_count, 1)
+        self.assertTrue(datetime.fromisoformat(api_response.time[:-1]))
+
+
+    def test_create_message_bad_request(self):
+        with self.assertRaises(ApiException) as context:
+            self.api_instance.create_message(self.account_id, self.invalid_message_request)
+            
+        self.assertEqual(context.exception.status, 400)
+
+        e = json.loads(context.exception.body)
+        self.assertEqual(e['type'], 'request-validation')
+        self.assertIsInstance(e['description'], str)
+        self.assertIsInstance(e['fieldErrors'], list)
+
+        field_error = e['fieldErrors'][0]
+        self.assertIsInstance(field_error['fieldName'], str)
+        self.assertIsInstance(field_error['description'], str)
```

Comment:
Would we get specific `fieldName` for this exception? If yes, can we check for the value instead of type?